tests/testthat/helper-orderly.server.R

```diff
@@ -125,6 +124,30 @@ orderly_path_orderly_run_rds <- orderly:::path_orderly_run_rds
 orderly_git_checkout_branch <- orderly:::git_checkout_branch
 ## nolint end
 
+orderly_git_example <- function(name, path = tempfile(), testing = FALSE) {
```

Comment:
Maybe we should move this into `orderly`? I can see updating `orderly_example` to have a `git` arg to initialise with git.

Reply:
yes, good idea. it's possibly where it came from in the first place?